### ISSUE TITLE
Enabling follow viewport on parallax background with mirroring on parallax layers causes lag in mirroring

### ISSUE THREAD

**Issue by digilena** [godotengine/godot#87317]
### Tested versions

-Reproducible in Godot Engine v4.0.3.stable.official.5222a99f5 and v4.2.1.stable.mono.official [b09f793f5]

### System information

Godot Engine v4.0.3.stable.official.5222a99f5 - https://godotengine.org OpenGL API 4.1 Metal - 83 - Compatibility - Using Device: Apple - Apple M2

### Issue description

When using the parallax background and parallax layer with mirroring turned on and follow viewport enabled on the parallax background, the mirroring becomes severely delayed when playing.

### Steps to reproduce

I have included my project with the relevant files and assets.
On the Parallax background layer, toggle between having the following viewport enabled/not enabled.
When not enabled the behavior of the mirroring works as expected, the background recreats automatically and without lag.
When enabled, the mirroring behavior does not work as expected, and there is a lag before it recreates itself.

### Minimal reproduction project (MRP)

[BackwardsIsland 2.zip](https://github.com/godotengine/godot/files/13970737/BackwardsIsland.2.zip)

**Comment by markdibarry** [comment]
I just took a look at your project and the issue you're having is that `follow_viewport` is a `CanvasLayer` property, not `ParallaxBackground`. It's turned off by default for both. Unlike `CanvasLayer` however, `ParallaxBackground`'s internal logic makes sure to follow your camera without it. It is a little confusing that turning off `follow_viewport` would cause the opposite to happen, but this is expected behavior, not a bug.

That being said, the PR I made recently (and is linked above) should make this process a little more straightforward if/when it gets merged, so feel free to show support if you're interested! In the meantime, I believe this is safe to close.
